src/core/gate.rs

```diff
@@ -163,96 +140,11 @@ impl Gate {
     }
 
     pub fn garbled(&self) -> Vec<S> {
```

Comment:
Why are these methods removed? They will be almost the same with FreeXOR in general, they will just add a separate branch with XOR/XNOR operations